### PR TITLE
Dont Catch Duplicated Pokemons option

### PR DESCRIPTION
**Pull request by plsfixrito**

### CODE REVIEW

File: PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs [hunk 1]
```diff
@@ -29,6 +29,12 @@ public static void Execute(Context ctx, StateMachine machine)
                     continue;
                 }
 
+                if (ctx.Inventory.GetPokemons().Result.Any(p => p.PokemonId == pokemon.PokemonId) && ctx.LogicSettings.DontCatchDuplicatedPokemon)
+                {
+                    Logger.Write("Already Have " + pokemon.PokemonId + " Skipped");
```
Comment (yamashi) [inline PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs]: No logger calls in Logic

Comment (plsfixrito) [inline PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs]: there is already a logger call before mine 
https://github.com/plsfixrito/NecroBot/blob/790a55f3e7997833b414acb7da80accc45b6a4bf/PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs#L25-L30

Comment (yamashi) [inline PoGo.NecroBot.Logic/Tasks/CatchNearbyPokemonsTask.cs]: Yep we will have to remove this one!

**Comment by Artud** [comment]
Remove logger call and submit new PR.

**Pull request closed without merging**